src/utils/media-url-utils.js

```diff
@@ -99,7 +99,8 @@ export const guessContentType = url => {
       matches[1];
     }
   }
-  const extension = new URL(url).pathname.split(".").pop();
+  const path = url.startsWith("/") ? url : new URL(url).pathname;
```

Comment:
`new URL(url, window.location).pathname` will work as well

Reply:
Ah yeah, I like that better.